### PR TITLE
fix toyball.js

### PR DESCRIPTION
**Pull request by EdgarPironti**

### CODE REVIEW

**Comment by hifi-gustavo** [comment]
An admin must verify this patch for the pull request build to fire. Type "ok to test" to begin testing this pull request or "add to whitelist" to test this and all future pull requests from this collaborator.

File: examples/controllers/hydra/toyball.js [hunk 1]
```diff
@@ -79,35 +71,35 @@ function checkControllerSide(whichSide) {
     var linearVelocity;
     var angularVelocity; 
     var AVERAGE_FACTOR = 0.33;
-    
+	var grabButtonPressed;
```
Comment (jherico) [inline examples/controllers/hydra/toyball.js]: Please modify your editor to use spaces, not tabs, and replace all the tabs in this file with spaces.

File: examples/controllers/hydra/toyball.js [hunk 2]
```diff
@@ -20,13 +20,8 @@ HIFI_PUBLIC_BUCKET = "http://s3.amazonaws.com/hifi-public/";
 // maybe we should make these constants...
 var LEFT_PALM = 0;
 var LEFT_TIP = 1;
```
Comment (jherico) [inline examples/controllers/hydra/toyball.js]: unused variable

File: examples/controllers/hydra/toyball.js [hunk 3]
```diff
@@ -20,13 +20,8 @@ HIFI_PUBLIC_BUCKET = "http://s3.amazonaws.com/hifi-public/";
...
-var LEFT_BUTTON_FWD = 5;
-var LEFT_BUTTON_3 = 3;
-
 var RIGHT_PALM = 2;
 var RIGHT_TIP = 3;
```
Comment (jherico) [inline examples/controllers/hydra/toyball.js]: unused variable

File: examples/controllers/hydra/toyball.js [hunk 4]
```diff
@@ -20,13 +20,8 @@ HIFI_PUBLIC_BUCKET = "http://s3.amazonaws.com/hifi-public/";
 // maybe we should make these constants...
 var LEFT_PALM = 0;
 var LEFT_TIP = 1;
-var LEFT_BUTTON_FWD = 5;
-var LEFT_BUTTON_3 = 3;
-
 var RIGHT_PALM = 2;
```
Comment (jherico) [inline examples/controllers/hydra/toyball.js]: unused variable

**Pull request merged**